### PR TITLE
Fixed erDiagram being whitespace sensitive where it shouldn't

### PR DESCRIPTION
**Pull request by Logerfo**

### CODE REVIEW

File: syntaxes/diagrams/erDiagram.yaml [hunk 1]
```diff
@@ -42,12 +42,12 @@
           name: keyword.control.mermaid
     - comment: '(entity) (relationship) (entity) : (label)'
       match: !regex |-
-        \s*([\w-]+)\s+ # entity
+        \s*([\w-]+)\s() # entity
```
Comment (bpruitt-goddard) [inline syntaxes/diagrams/erDiagram.yaml:45]: Should this be a capture group or should it be a `*` similar to the other changes?

Comment (Logerfo) [reply syntaxes/diagrams/erDiagram.yaml]: Oops, fixed!

**Comment by bpruitt-goddard** [comment]
Looks good. Can a test or two be added to the [associated test file](https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/blob/master/tests/diagrams/er.test.mermaid?short_path=734eb70)? Perhaps one of the [relationship lines](https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/blob/035024a621c95df5e5cdc9df1e1f00b436afd2af/tests/diagrams/er.test.mermaid#L36-L71) can be modified to reflect the change? Note you'll need to update the commented lines afterword to ensure the caret `^` symbols line up with the proper positions. You can then validate the tests via these commands (from the root of the project) `npm run convertYaml && npm test`

**Comment by Logerfo** [comment]
@bpruitt-goddard 
```console
C:\repos\vscode-mermaid-syntax-highlight  patch-1  ~1    20.5.0 ﴱ dev0 
npm run convertYaml && npm test

> mermaid-markdown-syntax-highlighting@1.5.3 convertYaml
> node build/ConvertYaml.js


> mermaid-markdown-syntax-highlighting@1.5.3 test
> vscode-tmgrammar-test 'tests/**/*.test.{mermaid,md}'

ERROR no test cases found
```

**Comment by bpruitt-goddard** [comment]
Any chance you are on windows? I think there is a bug on Windows [with the test library](https://github.com/PanAeon/vscode-tmgrammar-test/issues/1) that is fixed by changing the quotes on the glob pattern. Can you try making this change [here in package.json](https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/blob/035024a621c95df5e5cdc9df1e1f00b436afd2af/package.json#L61) and see if that picks up the tests?

```
"test": "vscode-tmgrammar-test \"tests/**/*.test.{mermaid,md}\""
```

**Comment by Logerfo** [comment]
@bpruitt-goddard what do you think?

**Pull request merged**